.github/CODEOWNERS

```diff
@@ -33,7 +33,7 @@
 /sdk/containerregistry/                             @pallavit @alzimmermsft @jianghaolu
 
 # PRLabel: %Azure.Core
-/sdk/core/                                          @alzimmermsft @jianghaolu @srnagar @hemanttanwar @anuchandy @lmolkova
+/sdk/core/                                          @alzimmermsft @jianghaolu @srnagar @anuchandy @lmolkova
```

Comment:
Add another line here for core amqp  - `/sdk/core/azure-core-amqp/` and add `@conniey @yijunxiems @ki1729 
 @srnagar @anuchandy` as owners.

Reply:
Done